meta-aos-rpi-domd/recipes-core/images/aos-image-rpi.bb

```diff
@@ -23,13 +23,15 @@ IMAGE_INSTALL:append = " \
 
 IMAGE_INSTALL:append = " \
     aos-messageproxy \
+    udev-rules \
 "
 
 PACKAGE_INSTALL:append = " \
     kernel-modules \
 "
 
 EXTRA_USERS_PARAMS = " \
+    groupadd --system gpio; \
```

Comment:
Should be in udev rules recipe. If you add this recipe group is added if not, group is not added.